R/Design-methods.R

```diff
@@ -4636,6 +4677,21 @@ setMethod("simulate",
           data = thisData
         )
 
+        # Get the MTD estimate from the samples.
+
+        target_dose_samples <- dose(
+          mean(object@nextBest@target),
+          model = object@model,
+          samples = thisSamples
+        )
+
+        # Create a function for additional statistical summary.
+        if (is.null(derive)) {
+          additional_stats <- list()
+        } else {
+          additional_stats <- lapply(derive, function(f) f(target_dose_samples))
+        }
```

Comment:
Same here.

Reply:
I think you need to add `derive = list()`to the function as done in line 212 above.